src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
... do you have any details on the compile failure? The only way I can see this failing is if somehow ```wq``` auto-resolved to ```nullptr_t```, but nothing that I can find is passing in ```nullptr``` (unless Clang is somehow substituting ```NULL``` for ```nullptr``` -- which is wrong since they are not the same)